modules/vpc/variables.tf

```diff
@@ -149,6 +149,8 @@ variable "security_groups" {
       - `type`: Specifies if rule will be evaluated on ingress (inbound) or egress (outbound) traffic.
       - `cidr_blocks`: List of CIDR blocks - for ingress, determines the traffic that can reach your instance. For egress
       Determines the traffic that can leave your instance, and where it can go.
+      - `ipv6_cidr_blocks`: List of IPv6 CIDR blocks - for ingress, determines the traffic that can reach your instance. For egress
+      Determines the traffic that can leave your instance, and where it can go. Defaults to null. 
```

Comment:
In variable description we have an example - can we add 1 rule with IPv6 to existing IPv4 rules ?